src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs

```diff
@@ -891,33 +891,32 @@ private void OnSuggestedActionsChanged(Workspace currentWorkspace, DocumentId cu
                 var provider = _owner;
                 using (var asyncToken = _owner.OperationListener.BeginAsyncOperation(nameof(GetSuggestedActionCategoriesAsync)))
                 {
-                    var selection = await GetSpanAsync(range).ConfigureAwait(false);
-                    if (selection != null)
+                    var document = range.Snapshot.GetOpenDocumentInCurrentContextWithChanges();
+                    using (var linkedTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken))
                     {
-                        var document = range.Snapshot.GetOpenDocumentInCurrentContextWithChanges();
-                        using (var linkedTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken))
+                        var linkedToken = linkedTokenSource.Token;
+
+                        var errorTask = Task.Run(
+                            () => GetFixLevelAsync(provider, document, range, linkedToken), linkedToken);
+
+                        var selection = await GetSpanAsync(range).ConfigureAwait(false);
+                        Task<string> refactoringTask = Task.FromResult((string)null);
+                        if (selection != null && requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.Refactoring))
```

Comment:
Does this code do the right thing if we ask for a different set of requested actions in the future? It's not an issue yet, but I could see the platform team wanting to add a new entry point to lightbulb in the future that requests only errors, or something similar.